### PR TITLE
[Core] Adding new mesh elements definitions

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Adding new mesh elements definitions.

**🆕 Changelog**

- [Adding new mesh elements definitions](https://github.com/KratosMultiphysics/Kratos/commit/c7920540b3d4ad13727c750744f6d7a191f0a2cf)

### CODE REVIEW

**Comment by philbucher** [comment]
@Igarizza you could use this in !12660

File: kratos/sources/kratos_application.cpp [hunk 1]
```diff
@@ -93,6 +93,20 @@ KratosApplication::KratosApplication(const std::string& ApplicationName)
       mElement3D20N( 0, GeometryType::Pointer(new Hexahedra3D20<NodeType >(GeometryType::PointsArrayType(20)))),
       mElement3D27N( 0, GeometryType::Pointer(new Hexahedra3D27<NodeType >(GeometryType::PointsArrayType(27)))),
 
+      mPointElement2D1N( 0, GeometryType::Pointer(new Point2D<NodeType >(GeometryType::PointsArrayType(1)))),
+      mPointElement3D1N( 0, GeometryType::Pointer(new Point2D<NodeType >(GeometryType::PointsArrayType(1)))),
```
Comment (rubenzorrilla) [inline kratos/sources/kratos_application.cpp]:
```suggestion
      mPointElement3D1N( 0, GeometryType::Pointer(new Point3D<NodeType >(GeometryType::PointsArrayType(1)))),
```

File: kratos/sources/kratos_application.cpp [hunk 2]
```diff
@@ -93,6 +93,20 @@ KratosApplication::KratosApplication(const std::string& ApplicationName)
...
+
+      mLineElement2D2N( 0, GeometryType::Pointer(new Line2D2<NodeType >(GeometryType::PointsArrayType(2)))),
+      mLineElement2D3N( 0, GeometryType::Pointer(new Line2D3<NodeType >(GeometryType::PointsArrayType(3)))),
+      mLineElement3D2N( 0, GeometryType::Pointer(new Line2D2<NodeType >(GeometryType::PointsArrayType(2)))),
```
Comment (rubenzorrilla) [inline kratos/sources/kratos_application.cpp]:
```suggestion
      mLineElement3D2N( 0, GeometryType::Pointer(new Line3D2<NodeType >(GeometryType::PointsArrayType(2)))),
```

File: kratos/sources/kratos_application.cpp [hunk 3]
```diff
@@ -160,26 +174,28 @@ void KratosApplication::RegisterKratosCore() {
     //Register specific conditions ( must be completed : conditions defined in kratos_application.h)
     //generic condition
     KRATOS_REGISTER_CONDITION("GenericCondition", mGenericCondition);
-    //point conditions
+    // Point conditions
```
Comment (rubenzorrilla) [inline kratos/sources/kratos_application.cpp:178]:
```suggestion
    
    // Point conditions
```
If we are to change it lets make it consistent with the ones below...

**Comment by loumalouomega** [comment]
This is ready @rubenzorrilla

**Review by rubenzorrilla — APPROVED** [review]

**Comment by Igarizza** [comment]
@rubenzorrilla @loumalouomega @philbucher thank you for the update!

**Pull request merged**